todo_app/app.py

```diff
@@ -1,11 +1,42 @@
 from flask import Flask
+from flask import render_template
+from flask import request 
+from flask import redirect
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
+from todo_app.data.session_items import get_item, save_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+   # items = get_items()    
+    #return render_template('index.html',items=items)
+    # used - https://note.nkmk.me/en/python-dict-list-sort/
```

Comment:
Generally, it's advised to prevent commented code from reaching the main branch. If you'll ever need it, you can get it from the git history